Code/Editor/AzAssetBrowser/AzAssetBrowserWindow.cpp

```diff
@@ -1057,10 +1057,49 @@ void AzAssetBrowserWindow::SetFavoritesWindowHeight(int height)
     m_ui->m_leftsplitter->setSizes(sizes);
 }
 
-void AzAssetBrowserWindow::SelectionChanged([[maybe_unused]] const QItemSelection& selected, [[maybe_unused]] const QItemSelection& deselected)
+void AzAssetBrowserWindow::SelectionChanged(const QItemSelection& selected, [[maybe_unused]] const QItemSelection& deselected)
 {
     OnFilterCriteriaChanged();
-}
 
+    // if we select 1 thing, give the previewer a chance to view it.
+    QModelIndexList selectedIndices = selected.indexes();
+
+    // Note that the selected indices might be different columns of the same rows.  Its still a valid "single selection"
+    // if there is only one unique row, even if there's more than 1 column
+    // we also don't care to actually count how many rows there are that are unique, we just need to know if there is exactly
+    // one row, so we can stop after finding more than one.
+
+    int numberOfuniqueRows = 0;
+    int lastSeenRow = -1;
+    for (const auto& element : selectedIndices)
```

Comment:
Its super annoying that QT does not have a "rows" call on this, as I believe there are other use case, maybe this could end up in the QTUtil.h in editor code as `ContainsOnlyOneRow` or similar

Reply:
I took a look at QtUtil and ugh, its a header-only template thing.  But I'll add it there.

Reply:
actually on second thought, the way I wrote this it only wants to know IF there's more than one row, not how many rowes there are.... I'll add that, but I don't know how useful it is in utils, and qtutil will be used in a bunch of other compile units that won't use it

Reply:
added to qtutil